### PR TITLE
Reactions!

### PR DESCRIPTION
**Pull request by LouieBloo**

### CODE REVIEW

File: src/app/components/modals/sound-effect-modal/sound-effect-modal.component.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    <div class="modal-box bg-slate-800 text-white absolute top-2">
+        <form method="dialog">
+            <button class="btn btn-sm btn-circle btn-ghost absolute right-2 top-2" id="closeSoundEffectModal">✕</button>
+        </form>
+
+        <h3 class="text-lg font-bold">Play Sounds</h3>
+        <p class="text-sm mb-4">Play sounds for all to hear! 15s cooldown.</p>
+
+        <div *ngFor="let group of getObjectKeys(soundList)">
```
Comment (domleggio) [inline src/app/components/modals/sound-effect-modal/sound-effect-modal.component.html]: this gets called constantly

File: src/app/components/effects/reactions/reactions.component.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<div *ngIf="showing">
```
Comment (domleggio) [inline src/app/components/effects/reactions/reactions.component.html:1]: extra div

File: src/app/components/effects/reactions/reactions.component.ts [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+  }
+
+  ngOnDestroy(): void {
+    this.clearTimeouts();
+    this.subscriptions.unsubscribe();
+  }
+
+  private clearTimeouts() {
+    if (this.showingTimeout) {
+      clearTimeout(this.showingTimeout);
```
Comment (domleggio) [inline src/app/components/effects/reactions/reactions.component.ts:149]: what is this clearTimeout?

**Review by domleggio — APPROVED** [review]

File: src/app/components/effects/reactions/reactions.component.ts (hunk 3 above)
Comment (LouieBloo) [reply src/app/components/effects/reactions/reactions.component.ts:149]: Its built in, it clears a setTimeout call

File: src/app/components/effects/reactions/reactions.component.html (hunk 2 above)
Comment (LouieBloo) [reply src/app/components/effects/reactions/reactions.component.html:1]: Its used, Id rather have a div at parent level than <picture>

File: src/app/components/modals/sound-effect-modal/sound-effect-modal.component.html (hunk 1 above)
Comment (LouieBloo) [reply src/app/components/modals/sound-effect-modal/sound-effect-modal.component.html]: I dont even know why I have that, removing

**Pull request merged**